docs/maintenance-mode.md

```diff
@@ -4,7 +4,14 @@ The repo includes a simple service unavailable page page that can be pushed to a
 
 When enabled, all requests will receive the [service unavailable page](/service_unavailable_page/web/public/internal/index.html).
 
-### Enable Maintenance mode
+## Instructions
```

Comment:
This is going to conflict with this i think https://github.com/DFE-Digital/register-trainee-teachers/pull/2847/files

Reply:
Sucks to be @gpeng 🤣